packages/data/README.md

```diff
@@ -227,45 +285,182 @@ registerStore( 'my-shop', {
+`Function`: A reducer that invokes every reducer inside the reducers object, and constructs a state object with the same shape.
 
-_Example:_
+### createRegistry
+
+[src/index.js#L16-L16](src/index.js#L16-L16)
+
+Creates a new store registry, given an optional object of initial store
+configurations.
+
+**Parameters**
+
+-   **storeConfigs** `Object`: Initial store configurations.
+
+**Returns**
+
+`WPDataRegistry`: Data registry.
+
+### createRegistryControl
+
+[src/index.js#L18-L18](src/index.js#L18-L18)
+
+Mark a control as a registry control.
+
+**Parameters**
+
+-   **registryControl** `function`: Function receiving a registry object and returning a control.
+
+**Returns**
+
+`function`: marked registry control.
+
+### createRegistrySelector
+
+[src/index.js#L18-L18](src/index.js#L18-L18)
+
+Mark a selector as a registry selector.
+
+**Parameters**
+
+-   **registrySelector** `function`: Function receiving a registry object and returning a state selector.
+
+**Returns**
+
+`function`: marked registry selector.
+
+### dispatch
+
+[src/index.js#L88-L88](src/index.js#L88-L88)
+
+Given the name of a registered store, returns an object of the store's action creators. Calling an action creator will cause it to be dispatched, updating the state value accordingly.
+
+**Usage**
 
 ```js
-function PriceDisplay( { price, currency } ) {
-	return new Intl.NumberFormat( 'en-US', {
-		style: 'currency',
-		currency,
-	} ).format( price );
-}
+const { dispatch } = wp.data;
 
-const { withSelect } = wp.data;
+dispatch( 'my-shop' ).setPrice( 'hammer', 9.75 );
+```
 
-const HammerPriceDisplay = withSelect( ( select, ownProps ) => {
-	const { getPrice } = select( 'my-shop' );
-	const { currency } = ownProps;
+**Parameters**
 
-	return {
-		price: getPrice( 'hammer', currency ),
-	};
-} )( PriceDisplay );
+-   **name** `string`: Store name
 
-// Rendered in the application:
-//
-//  <HammerPriceDisplay currency="USD" />
+**Returns**
+
+`Object`: Object containing the action creators.
+
+### plugins
+
+[src/index.js#L17-L17](src/index.js#L17-L17)
+
+Undocumented declaration.
+
+### registerGenericStore
+
+[src/index.js#L116-L116](src/index.js#L116-L116)
+
+Registers a generic store.
+
+**Parameters**
+
+-   **key** `string`: Store registry key.
+-   **config** `Object`: Configuration (getSelectors, getActions, subscribe).
+
+### registerStore
+
+[src/index.js#L126-L126](src/index.js#L126-L126)
+
+Registers a standard `@wordpress/data` store.
+
+**Parameters**
+
+-   **reducerKey** `string`: Reducer key.
+-   **options** `Object`: Store description (reducer, actions, selectors, resolvers).
+
+**Returns**
+
+`Object`: Registered store object.
+
+### RegistryConsumer
+
+[src/index.js#L14-L14](src/index.js#L14-L14)
+
+Undocumented declaration.
```

Comment:
This needs documenting. To be added as a JSDoc comment in the component.

Reply:
Are you considering this a blocker? Do you need help with it?

Reply:
Not a blocker from my point of view. It was undocumented and will stay that way after all.

I feel this is ready and is a good addition, but would like feedback on 1) did I miss something? 2) does the general approach works well for this package or are things we should improve?